plugins/cisco_umbrella_destinations/plugin.spec.yaml

```
@@ -0,0 +1,406 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: cisco_umbrella_destinations
+title: Cisco Umbrella Destinations
+version: 1.0.0
+supported_versions: ["2022-31-01"]
+description: Cisco Umbrella Destinations allows users to manage, block, and allow network destinations based on policies within their organization
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/cisco_umbrella_destinations
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.cisco.com
+tags:
+- cisco
+- umbrella
+- destinations
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [cisco, umbrella]
+  features: []
+connection:
+  api_key:
+    title: API Key
+    description: Cisco Umbrella Management API key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  api_secret:
+    title: API Secret Key
+    description: Cisco Umbrella Management API secret key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  organization_id:
+    title: Organization ID
+    description: ID for organization
+    type: integer
+    required: true
+    example: 1234567
+types:
+  meta:
+    destinationCount:
+      type: integer
+      description: Total number of destinations in a destination list
+      required: false
+    domainCount:
+      type: integer
+      description: Total number of domains in a destination list
+      required: false
+    urlCount:
+      type: integer
+      description: Total number of URLs in a destination list
+      required: false
+    ipv4Count:
+      type: integer
+      description: Total number of IP's in a destination list
```

Optional but just for consistency with the other descriptions
```suggestion
      description: Total number of IPs in a destination list
```